charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -26,6 +26,18 @@ metrics:
             {{- end }}
 {{- end }}
 
+{{- define "opentelemetry-collector.metrics.prometheus" -}}
+{{/* First replace env: with env! so we can split the host with the port and replace it back later */}}
+{{- $address:= .Values.config.service.telemetry.metrics.address | replace "{env:" "{env!" | split ":" }}
```

Comment:
cleverrrrr, i wish we could more easily test helm out here

Reply:
im trying to decide if this is better than checking for the index of the last : and then doing a substr on that though. I think this approach should always work, but it's maybe not as clear? Not sure... @TylerHelmuth thoughts?

Reply:
Im not fully understand how `regexFindAll` will help here. Because it will return the substrings not the index.
As far as i understand it would return `["${env","address}","8888"]` or `["address","8888"]`
so reacting on it would be stange.

I had the idear of getting the last index of `:` to. But there is no `findLast` and `index` is not working on strings. Not even `reverse` is working on strings...

Reply:
This logic is temporary so I am fine with this approach. Looking through the string functions I don't see another way to do this cleanly.